### PR TITLE
Correct p5p package name

### PR DESCRIPTION
**Pull request by scotthain**
### Description

Briefly describe the new feature or fix here

--------------------------------------------------
/cc @chef/omnibus-maintainers <- This ensures the Omnibus Maintainers team are notified to review this PR.

@chef/engineering-services

### CODE REVIEW

**Comment by yzl** [comment]
👍 pending tests

File: lib/omnibus/packagers/ips.rb [hunk 1]
```diff
@@ -81,7 +81,16 @@ def publisher_prefix(val = NULL)
...
     def package_name
-      "#{safe_base_package_name}.p5p"
+      "#{omnibus_package_name}.p5p"
+    end
+
+    #
+    # We want to have a consistent omnibus package name, but we are required
+    # to have the fmri version for the IPS repo creation.
+    #
+    def omnibus_package_name
```
Comment (schisamo) [inline lib/omnibus/packagers/ips.rb]: What is the purpose of introducing an extra `omnibus_package_name` method here? We should also include an iteration in the package name like we do for all other packages.

Comment (scotthain) [inline lib/omnibus/packagers/ips.rb]: Thanks - I missed the iteration!

The reason for having a new method is that the fmri based naming for IPS repo based packages is (for example) `chef@12.13.21,5.11-1`. It's required to name it as such for the packaging commands to work. I didn't think we wanted to name the package `chef@12.13.21,5.11-1.p5p` (tho we could). There is no 'standard' for what the p5p package looks like and usually they are shipped as like `chef.p5p`, which is obviously not going to work for us. I added the `omnibus_package_name` method to be consistent with the other omnibus package output, while still allowing for the packaging commands to function with the fmri input.

Comment (schisamo) [inline lib/omnibus/packagers/ips.rb]: @scotthain Shouldn't that come from the existing `fmri_package_name` method?

**Comment by scotthain** [comment]
Verified locally ready for final review @chef/engineering-services

**Comment by schisamo** [comment]
👍

**Pull request merged**